instrumentation/netty/netty-4-common/library/src/main/java/io/opentelemetry/instrumentation/netty/v4/common/internal/client/NettyInstrumentationFlag.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.netty.v4.common.internal.client;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public enum NettyInstrumentationFlag {
+  ENABLED,
+  ERROR_ONLY,
+  DISABLED;
+
+  public static NettyInstrumentationFlag enabledOrErrorOnly(boolean b) {
```

Comment:
this method name confused me, but I can't think of a better one...

Reply:
I couldn't either - if anyone has a suggestion, I'm all ears